deploy/scripts/start-service.sh

```diff
@@ -49,7 +49,7 @@ docker run \
     --network openregisters \
     --log-driver=fluentd \
     --log-opt fluentd-async-connect=true \
-    openjdk:8-jre-alpine \
```

Comment:
have you deliberately chosen to move away from the alpine-based image? `8u131-jre-alpine` is available